redbot/core/bank.py

```diff
@@ -332,6 +332,26 @@ def _invalid_amount(amount: int) -> bool:
         await _conf.clear_all_members(guild)
 
 
+async def bank_local_clean(guild: discord.Guild, member: discord.member = None):
+    """Remove data for members who are no longer in the server"""
```

Comment:
You need to document what this does better. This doc does not even state that it can also be used to delete a specific member's bank account.